src/bindgen/language_backend/clike.rs

```diff
@@ -913,23 +913,34 @@ impl LanguageBackend for CLikeLanguageBackend<'_> {
                 }
 
                 write!(out, "{{ ");
-                let mut is_first_field = true;
                 // In C++, same order as defined is required.
                 let ordered_fields = out.bindings().struct_field_names(path);
-                for ordered_key in ordered_fields.iter() {
+                let last_i = ordered_fields.len() - 1;
```

Comment:
Ah, the question didn't get saved. It's basically... can't this underflow? It'd be better to do `if ordered_fields.is_empty() { 0 } else { ordered_fields.len() - 1 };`